src/app/@ansyn/ansyn/api/ansyn-api.service.ts

```diff
@@ -118,6 +124,13 @@ export class AnsynApi {
 		this.store.dispatch(new DisplayOverlayAction({ overlay, mapId: this.activeMapId, forceFirstDisplay: true }));
 	}
 
+	displayOverlayOnMap(overlay: IOverlay, mapNo: number): void {
```

Comment:
make mapNo optional and if undefined display on active map (remove the other function)
mapNo -> mapNumber

Reply:
done